categories/categories-router.js

```diff
@@ -12,7 +12,7 @@ router.post('/:id/categories', restricted, (req, res) => {
   const issue = {...req.body, user_id: id}
 
   Categories.add(issue)
-    .then(category => {
+    .then(res => {
```

Comment:
This is easier to read and understand using 'res' instead of 'category'. Good call!